features/org.jboss.tools.vpe.feature/feature.xml

```diff
@@ -19,10 +19,6 @@
       %license
    </license>
 
-   <includes
-         id="org.jboss.tools.vpe.preview.feature"
```

Comment:
Why do we need to remove vpe.preview.feature from vpe feature? PRM feature could include vpe.preview but why we need to modify vpe feature?